lib/msf/core/auxiliary/report.rb

```diff
@@ -115,7 +115,7 @@ def get_host(opts)
   # Report a client connection
   #
   # opts must contain
-  #	:host      the address of the client connecting
+  #	:host the address of the client connecting
```

Comment:
As long as we're fixing docs, these should all be converted to the `@option` format, e.g.:
```
# @option opts [String] :host the address of the client connecting
```